02_configure_host.sh

```diff
@@ -85,13 +85,19 @@ if [ ${NUM_EXTRA_WORKERS} -ne 0 ]; then
   jq "{nodes: .nodes[-${NUM_EXTRA_WORKERS}:]}" ${ORIG_NODES_FILE} | tee ${EXTRA_NODES_FILE}
 fi
 
+ZONE="\nZONE=libvirt"
+sudo systemctl enable --now firewalld
+
 # Allow local non-root-user access to libvirt
 # Restart libvirtd service to get the new group membership loaded
 if ! id $USER | grep -q libvirt; then
   sudo usermod -a -G "libvirt" $USER
```

Comment:
Btw this won't work, it requires a `newgrp` call to take effect.

Reply:
If it doesn't work it's unrelated to the CI problem, but the restart of the libvirt daemon based on the comment seems to be enough.  The libvirt daemon verifies the users group membership I guess, not virsh running in the local shell.

Reply:
Quite possibly. But yeah, it's unrelated here.